action/src/main/kotlin/com/samples/pusher/core/SamplesPusher.kt

```diff
@@ -8,11 +8,11 @@ interface SamplesPusher {
   fun configure(fn: PusherConfiguration.() -> Unit): SamplesPusherImpl
 
   /**
-   * @return true if all is ok
+   * @return true if all is ok and bad samples are absent
```

Comment:
imho I always expect the Boolean return type when I see the method with prefix "is..." of something similar